### PR TITLE
OP-1184 New inventory for all products or for specific product

### PR DESCRIPTION
**Pull request by ArnaudFonzam**
See OP-1184.

### CODE REVIEW

File: sql/step_a104_create_table_inv_and_invr.sql [hunk 1]
```diff
@@ -19,7 +19,7 @@ CREATE TABLE OH_MEDICALDSRINVENTORY (
 
 CREATE TABLE OH_MEDICALDSRINVENTORYROW (
 	MINVTR_ID int NOT NULL AUTO_INCREMENT,
-	MINVTR_THEORETIC_QTY float  NOT NULL default 0,
+	MINVTR_THEORITIC_QTY float  NOT NULL default 0,
```
Comment (mwithi) [inline sql/step_a104_create_table_inv_and_invr.sql:22]: "THEORETIC" is the correct spelling

File: sql/load_demo_data.sql [hunk 2]
```diff
@@ -2749,6 +2749,28 @@ INSERT INTO `oh_medicaldsr` (`MDSR_ID`, `MDSR_MDSRT_ID_A`, `MDSR_CODE`, `MDSR_DE
...
+--
+
+LOCK TABLES `oh_medicalinventoryrow` WRITE;
+/*!40000 ALTER TABLE `oh_medicaldsrinventoryrow` DISABLE KEYS */;
+INSERT INTO `oh_medicaldsrinventoryrow`(`MINVTR_ID`, `MINVTR_THEORITIC_QTY`, `MINVTR_REAL_QTY`, `MINVTR_INVT_ID`, `MINVTR_MDSR_ID`, `MINVTR_LT_ID_A`, `MINVTR_LOCK`, `MINVTR_CREATED_BY`, `MINVTR_CREATED_DATE`, `MINVTR_LAST_MODIFIED_BY`, `MINVTR_LAST_MODIFIED_DATE`, `MINVTR_ACTIVE`) VALUES (1,0,0,1,103,'-1554622624438837880',1,NULL,NULL,NULL,NULL,1);
+INSERT INTO `oh_medicaldsrinventoryrow`(`MINVTR_ID`, `MINVTR_THEORITIC_QTY`, `MINVTR_REAL_QTY`, `MINVTR_INVT_ID`, `MINVTR_MDSR_ID`, `MINVTR_LT_ID_A`, `MINVTR_LOCK`, `MINVTR_CREATED_BY`, `MINVTR_CREATED_DATE`, `MINVTR_LAST_MODIFIED_BY`, `MINVTR_LAST_MODIFIED_DATE`, `MINVTR_ACTIVE`) VALUES (2,0,0,1,104,'-1769145966120870973',1,NULL,NULL,NULL,NULL,1);
+INSERT INTO `oh_medicaldsrinventoryrow`(`MINVTR_ID`, `MINVTR_THEORITIC_QTY`, `MINVTR_REAL_QTY`, `MINVTR_INVT_ID`, `MINVTR_MDSR_ID`, `MINVTR_LT_ID_A`, `MINVTR_LOCK`, `MINVTR_CREATED_BY`, `MINVTR_CREATED_DATE`, `MINVTR_LAST_MODIFIED_BY`, `MINVTR_LAST_MODIFIED_DATE`, `MINVTR_ACTIVE`) VALUES (3,0,0,1,415,'-4111916415666582009',1,NULL,NULL,NULL,NULL,1);
```
Comment (mwithi) [inline sql/load_demo_data.sql:2770]: "THEORETIC" is the correct spelling

File: src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java [hunk 3]
```diff
@@ -56,7 +56,7 @@ public MedicalInventory newMedicalInventory(MedicalInventory medicalinventory) t
 	 * @return the updated {@link MedicalInventory} object.
 	 * @throws OHServiceException
 	 */
-	public MedicalInventory updateMedicalInventoryRow(MedicalInventory medicalInventory) throws OHServiceException {
+	public MedicalInventory updateMedicalInventory(MedicalInventory medicalInventory) throws OHServiceException {
```
Comment (mwithi) [inline src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java:59]: Why this change if the service method called is `updateMedicalInventoryRow`?

File: src/main/java/org/isf/medicals/service/MedicalsIoOperations.java [hunk 4]
```diff
@@ -54,13 +54,23 @@ public class MedicalsIoOperations
 	
 	/**
 	 * Retrieves the specified {@link Medical}.
-	 * @param code the medical code
+	 * @param code the medical id.
```
Comment (mwithi) [inline src/main/java/org/isf/medicals/service/MedicalsIoOperations.java:57]: the correct param name is `code`

**Review by mwithi — CHANGES_REQUESTED** [review]
Some comments.
If the changes are meant to be shared with OP-1200, this PR should be toward develop branch, shouldn't they?

File: src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java (hunk 3 above)
Comment (ArnaudFonzam) [reply src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java:59]: Because the right name is `updateMedicalInventory ` not `updateMedicalInventoryRow` we already have `updateMedicalInventoryRow` on `MedicalInventoryRowManager`

Comment (mwithi) [reply src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java:59]: ok so you have to change also the called method on ioOperations?
